src/protocols/secure_channel/CASESession.cpp

```diff
@@ -208,11 +208,12 @@ CHIP_ERROR CASESession::Init(uint16_t localSessionId, SessionEstablishmentDelega
 }
 
 CHIP_ERROR
-CASESession::ListenForSessionEstablishment(uint16_t localSessionId, FabricTable * fabrics, SessionEstablishmentDelegate * delegate,
+CASESession::ListenForSessionEstablishment(SessionHolder & secureSessionHolder, FabricTable * fabrics,
```

Comment:
Again, should probably be `SessionHolder &&`.

Reply:
Also superseded.